README.md

```diff
@@ -102,6 +102,11 @@ optional arguments:
   --keep KEEP           number of top solutions kept (must be equal or less than --jobs/-j)
   --strategy STRATEGY   one of "best-wins", "worst-wins", "extreme-wins"
   --cluster CLUSTER     dask cluster address (defaults to local cluster)
+  -t TEMP, --temp TEMP  starting temperature for the simulated annealing
+  -e TEMP, --endtemp TEMP
+                        ending temperature for the simulated annealing
+  -a AUTOTEMP --auto_temp 
```

Comment:
This is counter-intuitive; please change it to take an argument: 0 for off, 1 for on (default 1).